policyengine_us/variables/gov/states/de/tax/income/subtractions/exclusions/de_elderly_or_disabled_income_exclusion.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_us.model_api import *
+
+
+class de_elderly_or_disabled_income_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware aged or disabled exclusion"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.DE
+
+    def formula(tax_unit, period, parameters):
+        # First get their filing status.
+        filing_status = tax_unit("filing_status", period)
+        p = parameters(
+            period
+        ).gov.states.de.tax.income.subtractions.exclusions.persons_60_or_over_or_disabled
+
+        # Get the individual disabled status.
+        disabled_head = tax_unit("disabled_head", period)
+        disabled_spouse = tax_unit("disabled_spouse", period)
+
+        # Get the individual filer's age and eligibility.
+        age_head = tax_unit("age_head", period)
+        age_head_eligible = (age_head >= p.threshold.age).astype(int)
+
+        # Get spouse age and eligibility
+        age_spouse = tax_unit("age_spouse", period)
+        age_spouse_eligible = (age_spouse >= p.threshold.age).astype(int)
+
+        # Get the tax unit income
+        is_joint = tax_unit("tax_unit_is_joint", period)
```

Comment:
```suggestion
        earned_income = tax_unit("tax_unit_earned_income", period)
```

Reply:
@MaxGhenis Would you mind confirming if we don't need to check the filing status for this? According to the tax law (https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf, page8 Line 11), if the filers are filling JOINTLY, we need to combine both the head and the spouse's income to check if the amount exceeds the threshold.

Reply:
We do not need to check filing status. Only joint tax unit totals include spouse income.